extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -903,7 +903,11 @@ export class AmpLightboxGallery extends AMP.BaseElement {
   transitionImg_(sourceElement, enter) {
     return this.getCurrentElement_().imageViewer.getImpl()
         .then(imageViewer => {
-          const {width, height} = imageViewer.getImageBoxWithOffset();
+          const imageBoxWithOffset = imageViewer.getImageBoxWithOffset();
```

Comment:
Nit: The old code will compress better.

```suggestion
          const {width, height} = imageViewer.getImageBoxWithOffset() || {};
```